resources/views/admin/createuser.blade.php

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Move toàn bộ nội dung của file này vào `resources/views/layouts/admin/master.blade.php` cho anh nhé